src/kademlia/dht_state.cpp

```diff
@@ -40,12 +40,40 @@ namespace dht
 {
 namespace
 {
-	node_id extract_node_id(bdecode_node const& e, string_view key)
+	node_ids_t extract_node_ids(bdecode_node const& e, string_view key)
```

Comment:
I wonder if it would make sense to give this function external linkage, so we could have a test for it.